### DISCUSSION TITLE
No bottle available for my published Tap causing failed installation on M1 Mac. Works on Intel Mac.

### DISCUSSION THREAD

**Discussion by solidiquis** [Homebrew/discussions#3249] [Casks] [answered]
I recently published this new repo for my tabs: https://github.com/solidiquis/homebrew-tap

There's currently only one formula in there. On my Intel mac, doing the following leads to successful installation:
```
$ brew tap solidiquis/erdtree
$ brew install erdtree
```

Doing that exact same thing on my M1 mac yields the following error:
```
$ brew install erdtree
Error: erdtree: no bottle available!
You can try to install from source with:
  brew install --build-from-source erdtree
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

I'd rather not have people build-from-source for convenience. Any suggestions?

**Comment by Bo98** [comment, marked as answer]
This should only happen if it detects it living in the Homebrew/core tap. What does `brew install --debug erdtree` output?

**Reply by solidiquis** [reply]
This is my output:
```
$ brew install --debug erdtree
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/erdtree.rb
Error: erdtree: no bottle available!
You can try to install from source with:
  brew install --build-from-source erdtree
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

**Reply by Bo98** [reply]
Yes it does seem to be what I suspect. It's loading `/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/erdtree.rb`. I'm guessing this is a local file you had?

**Reply by solidiquis** [reply]
@Bo98 What's the best course of action here?

I added this via `$ brew tap solidiquis/erdtree`.

On my Intel Mac it's going to homebrew-tap as opposed to homebrew-core with the same `$ brew tap solidiquis/erdtree` command.

**Reply by Bo98** [reply]
You must have created that file yourself - you can delete it if you don't want it anymore. `brew tap solidiquis/erdtree` does not touch homebrew-core.

**Reply by solidiquis** [reply]
@Bo98 actually.. deleting that file magically fixed everything. Thank you!!!
